ExchangeRateUpdater/ExchangeRateProvider.cs

```diff
@@ -13,7 +70,26 @@ public class ExchangeRateProvider
         /// </summary>
         public IEnumerable<ExchangeRate> GetExchangeRates(IEnumerable<Currency> currencies)
         {
-            return Enumerable.Empty<ExchangeRate>();
+            if (exchangeRates.Count == 0)
```

Comment:
Sorry, that's our extension method, it's implementation is what should be used - `!Any()`